javascript-local-storage/data.js

```diff
@@ -0,0 +1,13 @@
+/* exported todos */
+
+let todos = [];
+
+function beforeUnload(event) {
+  const todosJSON = JSON.stringify(todos);
+  localStorage.setItem('javascript-local-storage', todosJSON);
+}
+
+window.addEventListener('beforeunload', beforeUnload);
+
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
This should be inside an `if` statement. Checking if `previousTodosJSON` is null or not